docs/redirects.js

```diff
@@ -162,8 +162,8 @@ const misc = [
     permanent: true,
   },
   {
-    source: '/whats_new',
-    destination: '/roadmap/whats_new',
+    source: '/roadmap/whats_new',
+    destination: '/whats_new',
```

Comment:
Switching these back, in case people made bookmarks already

Reply:
_sigh_ this is why I wanted the flattest URL structure possible…hopefully we don't accumulate too many more of these in the future